drivers/hmis/app/graphql/mutations/update_project.rb

```diff
@@ -20,17 +20,21 @@ def resolve(id:, input:, confirmed:)
     end
 
     def create_errors(project, input)
-      return [] unless project.operating_end_date.blank? && input.operating_end_date.present?
-
       errors = HmisErrors::Errors.new
 
-      funder_count = project.funders.where(end_date: nil).count
-      inventory_count = project.inventories.where(inventory_end_date: nil).count
-      open_enrollments = Hmis::Hud::Enrollment.open_on_date.in_project_including_wip(project.id, project.project_id)
-
-      errors.add :base, :information, severity: :warning, full_message: "Project has #{open_enrollments.count} open #{'enrollment'.pluralize(open_enrollments.count)}." if open_enrollments.present?
-      errors.add :base, :information, severity: :warning, full_message: "#{funder_count} open #{'funder'.pluralize(funder_count)} will be closed." if funder_count.positive?
-      errors.add :base, :information, severity: :warning, full_message: "#{inventory_count} open inventory #{'record'.pluralize(inventory_count)} will be closed." if inventory_count.positive?
+      # If project end date is changing
```

Comment:
I think based on your suggestion @eanders this is something we should always warn about if the end date is changed.
Previously it was only flagged if the project was being "closed for the first time" (end date went from nil->value)

Reply:
That make sense.  I doubt OperatingEndDates change very often once set, but I could see someone adding the wrong year and then correcting it.  No harm in checking whenever it is changed (and present.)